packages/badge/src/Badge.ts

```diff
@@ -14,102 +14,24 @@ import {
     CSSResultArray,
     html,
     nothing,
-    SizedMixin,
-    SpectrumElement,
     TemplateResult,
 } from '@spectrum-web-components/base';
-import { property } from '@spectrum-web-components/base/src/decorators.js';
 
-import { ObserveSlotText } from '@spectrum-web-components/shared/src/observe-slot-text.js';
-import { ObserveSlotPresence } from '@spectrum-web-components/shared/src/observe-slot-presence.js';
+import { BadgeBase } from './Badge.base.js';
 import styles from './badge.css.js';
 
-export const BADGE_VARIANTS = [
-    'accent',
-    'neutral',
-    'informative',
-    'positive',
-    'negative',
-    'notice',
-    'fuchsia',
-    'indigo',
-    'magenta',
-    'purple',
-    'seafoam',
-    'yellow',
-    'gray',
-    'red',
-    'orange',
-    'chartreuse',
-    'celery',
-    'green',
-    'cyan',
-    'blue',
-] as const;
-export type BadgeVariant = (typeof BADGE_VARIANTS)[number];
-export const FIXED_VALUES = [
-    'inline-start',
-    'inline-end',
-    'block-start',
-    'block-end',
-] as const;
-export type FixedValues = (typeof FIXED_VALUES)[number];
```

Comment:
This update will result in a _BREAKING CHANGE_ for consumers who are importing directly from `Badge.ts`. To keep our downstream teams unblocked while we work toward the planned architecture cleanup, let’s re-export these constants from the current module. This way, we can maintain backward compatibility in the short term.
In `Badge.ts` please add this above any other code which uses these symbols
```ts
export { BADGE_VARIANTS, BadgeVariant, FIXED_VALUES, FixedValues } from './Badge.base.js';
```